website/content/en/preview/getting-started/scripts/add-provisioner.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -euo pipefail #fail if one step fails
+
+if [ "$#" -ne 1 ]
+then
+  echo "Missing required Karpenter version. Usage: install.sh v0.0.1"
+  exit 1
+fi
+
+export KARPENTER_VERSION=$1
+
+declare -a steps=(
+  step01-config.sh
+  step12-add-provisioner.sh
+  step13-automatic-node-provisioning.sh
+  step14-automatic-node-termination.sh
```

Comment:
I think adding a node-provision command followed by an immediate node-termination command might be a little confusing for users as it'll scale up and then instantly scale down, seeing a node provision then immediately be considered underutilized. 

Consider putting the deprovisioning commands as comments below, and renaming this to `provision-node.sh`

Reply:
good find, I moved it to clean-up instead.